packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/form/ViewFormDescriptionConverterSwitch.java

```diff
@@ -23,6 +23,7 @@
 
```

Comment:
In `ViewFormDescriptionEditorConverterSwitchTests#callGetWidgetLabel`, the call of `ViewFormDescriptionEditorConverterSwitch` seems not been updated with the new parameter.

Reply:
It is, but might have been included in the wrong commit initialy.